jest-common/src/main/java/io/searchbox/client/JestClient.java

```diff
@@ -14,7 +14,7 @@
 
     <T extends JestResult> T execute(Action<T> clientRequest) throws IOException;
 
-    <T extends JestResult> void executeAsync(Action<T> clientRequest, JestResultHandler<T> jestResultHandler);
+    <T extends JestResult> void executeAsync(Action<T> clientRequest, JestResultHandler<? super T> jestResultHandler);
```

Comment:
What's the reason for this (as `T` is already defined as `T extends JestResult`)?